config/opal_check_pc.m4

```diff
@@ -0,0 +1,40 @@
+dnl -*- shell-script -*-
+dnl
+dnl Copyright (c) 2021 IBM Corporation.  All rights reserved.
+dnl
+dnl $COPYRIGHT$
+dnl
+dnl Additional copyrights may follow
+dnl
+dnl $HEADER$
+dnl
+
+AC_DEFUN([OPAL_CHECK_LFLAGS], [
+         AC_MSG_CHECKING([for pc file for])
+         AC_MSG_RESULT([$1])
+
+	 num_paths=$#
```

Comment:
I think you have a tab on this one line, because your indenting doesn't line up

Reply:
Got it.